wapitiCore/attack/mod_cookieflags.py

```diff
@@ -0,0 +1,42 @@
+import requests
+from wapitiCore.attack.attack import Attack
+from wapitiCore.net.web import Request
+from wapitiCore.language.vulnerability import Additional
+
+class mod_cookieflags(Attack):
+
+    name = "cookieflags"
+
+    def check_secure_flag(self, cookie: object):
+
+        return cookie.secure
+
+    def check_httponly_flag(self, cookie: object):
+
+        return 'HttpOnly' in cookie._rest
+
+    def attack(self):
+        url = self.persister.get_root_url()
+        session = requests.session()
+        _req = session.get(url)
+        request = Request(url)
+        for cook in session.cookies:
+            self.log_blue("Check HttpOnly flag :")
```

Comment:
You should log the cookie name otherwise we don't know which one is wrong : 
```
./bin/wapiti -m cookieflags -u https://securite-info.eu/ -v2  --scope url

     __    __            _ _   _ _____
    / / /\ \ \__ _ _ __ (_) |_(_)___ /
    \ \/  \/ / _` | '_ \| | __| | |_ \ 
     \  /\  / (_| | |_) | | |_| |___) |
      \/  \/ \__,_| .__/|_|\__|_|____/
                  |_|                 
Wapiti-3.0.3 (wapiti.sourceforge.io)
[+] GET https://securite-info.eu/ (0)
[*] Enregistrement de l'état du scan, veuillez patienter...

 Note
========
Ce scan a été sauvé dans le fichier /home/devloop/.wapiti/scans/securite-info.eu_url_632bf5ff.db
[*] Wapiti a trouvé 1 URLs et formulaires lors du scan
[*] Chargement des modules :
         backup, blindsql, buster, cookieflags, crlf, delay, exec, file, htaccess, methods, nikto, permanentxss, redirect, shellshock, sql, ssrf, wapp, xss, xxe

[*] Lancement du module cookieflags
Check HttpOnly flag :
Secure flag is not set
Check HttpOnly flag :
Secure flag is not set
```

Something like this would be better

```
Checking cookie __cfuid
Secure flag is not set
Checking cookie SERVERID112363
Secure flag is not set
Checking cookie Whatever
HttpOnly flag is not set
```

Reply:
Fixed